vic/drivers/shared_image/src/vic_restore.c

```diff
@@ -0,0 +1,1011 @@
+        for (i = 0; i < local_domain.ncells_active; i++) {
+            all_vars[i].lake_var.volume = dvar[i];
+        }
+
+        // lake layer temperatures: lake_var.temp[nidx]
+        for (j = 0; j < options.NLAKENODES; j++) {
+            d3start[0] = j;
+            get_scatter_nc_field_double(filenames.init_state,
+                                        "Lake_layer_temp",
+                                        d3start, d3count, dvar);
+            for (i = 0; i < local_domain.ncells_active; i++) {
+                all_vars[i].lake_var.temp[j] = dvar[i];
+            }
+        }
+
+        // vertical average lake temperature: lake_var.tempavg
+        get_scatter_nc_field_double(filenames.init_state,
+                                    "Lake_average_temp",
+                                    d2start, d2count, dvar);
+        for (i = 0; i < local_domain.ncells_active; i++) {
+            all_vars[i].lake_var.tempavg = dvar[i];
+        }
+
+        // lake ice area fraction: lake_var.areai
+        get_scatter_nc_field_double(filenames.init_state,
+                                    "Lake_ice_area_frac",
+                                    d2start, d2count, dvar);
+        for (i = 0; i < local_domain.ncells_active; i++) {
+            all_vars[i].lake_var.areai = dvar[i];
+        }
+
+        // new lake ice area fraction: lake_var.new_ice_area
+        get_scatter_nc_field_double(filenames.init_state,
+                                    "Lake_ice_area_frac_new",
+                                    d2start, d2count, dvar);
+        for (i = 0; i < local_domain.ncells_active; i++) {
+            all_vars[i].lake_var.new_ice_area = dvar[i];
+        }
+
+        // lake ice water equivalent: lake_var.ice_water_eq
+        get_scatter_nc_field_double(filenames.init_state,
+                                    "Lake_ice_water_equivalent",
+                                    d2start, d2count, dvar);
+        for (i = 0; i < local_domain.ncells_active; i++) {
+            all_vars[i].lake_var.ice_water_eq = dvar[i];
+        }
+
+        // lake ice height: lake_var.hice
+        get_scatter_nc_field_double(filenames.init_state,
+                                    "Lake_ice_height",
+                                    d2start, d2count, dvar);
+        for (i = 0; i < local_domain.ncells_active; i++) {
+            all_vars[i].lake_var.hice = dvar[i];
+        }
+
+        // lake ice temperature: lake_var.tempi
+        get_scatter_nc_field_double(filenames.init_state,
+                                    "Lake_ice_temp",
+                                    d2start, d2count, dvar);
+        for (i = 0; i < local_domain.ncells_active; i++) {
+            all_vars[i].lake_var.tempi = dvar[i];
+        }
+
+        // lake ice snow water equivalent: lake_var.swe
+        get_scatter_nc_field_double(filenames.init_state,
+                                    "Lake_ice_snow_water_equivalen",
```

Comment:
Are we really saving one character by shaving off the final `t` or is this just a typo. Either way, please add the `t`